### ISSUE TITLE
Adding a "Evict Tenant" button to /companies

### ISSUE THREAD

**Issue by ItzBrittanyBitxh** [InZidiuZ/op-framework-issue-tracker#1998]
**Assurance:**
 Yesss

**Summary:**

Adding a "Evict Tenant" Button to the Company tablet for Realestate owners. Currently the only way someone is evicted form an address is when they stop paying rent. And when they do stop paying rent, the realtor eats the cost of the week they left, without any way to get it back.

OR 

Changing the "overdue" time for eviction from 7 days to 3-4 Days - to allow for faster RP with how the city is moving and a more reliable system in place.  - same reasoning.

**Reason:**
Currently if someone fresh to the city comes in and buys a $2k apartment, they will pay 2k up front, and if they decide to not come back to the city, that apartment is now owned and unable to be resold for 14 days from purchase, which in the RP world is a LONG time. 

If added, this would mainly be used for people who have given notice that they are not going to continue rent (eg. moving to a new place) OR if a realtor wants to set their own timeframe for eviction eg. "Here is the keys, keep in mind with this company, if you go more then 4 days overdue, you will be evicted" - Creates more RP and a much more manageable system for Real estate.

**Comment by Masik52** [comment]
Sweet baby Jesus yes to this.  There is nothing worse than having to wait on a property to default when the tenant has told you they are letting it go. If the concern is it’s something that a realtor could abuse they could have their company taken from them through court RP or by the staff team.

**Comment by Sinister-Neonfire** [comment]
I'd be a bit concerned if a realtor could just kick me out at any point. Perhaps only allow them to evict if the rent is overdue? It would also be nice for renters to have the option to voluntarily release the property back to realtor.

**Comment by AdeptCL** [comment]
I'm not even a realtor, but I do massively agree with this.
I believe that this will allow a potential for court RP to take place. Or, it will help the realtors with how @ItzBrittanyBitxh mentioned:

> Currently if someone fresh to the city comes in and buys a $2k apartment, they will pay 2k up front, and if they decide to not come back to the city, that apartment is now owned and unable to be resold for 14 days from purchase, which in the RP world is a LONG time.

**Comment by claymstehling** [comment]
oh I as a none realtor love this so much! I would also love to see it implemented to where you can transfer rentorship of the house to someone else if needed.

**Comment by Nuziatella** [comment]
If this option is added, how does one prevent abuse? 
Someone could for example just evict all their tenants and take items that are inside the homes. If something like this was added then all inventories would be deleted and no longer accessible if a tenant is force evicted.

**Comment by InZidiuZ** [comment]
Closing as outdated. If this is still relevant, please create a new issue.
